src/vs/platform/contextkey/test/common/contextkey.test.ts

```diff
@@ -376,4 +376,10 @@ suite('ContextKeyExpr', () => {
 			)!
 		));
 	});
+
+	// --- Start Positron ---
+	test('Positron Context Key', () => {
+		assert.ok(ContextKeyExpr.has('isPositron').equals(ContextKeyExpr.true()));
+	});
+	// --- End Positron ---
```

Comment:
Agreed this should be enough to verify the context key works for now 👍 

If we'd like to test in an extension, one idea is to add a command to positron-zed which has `enablement` based on the Positron context? Then we can verify that the context key is working if the command is available in the Command Prompt.

`extensions/positron-zed/package.json` (I think something like this might work, but untested!)

```json
            {
                "category": "Zed",
                "command": "zed.checkIsPositronContext",
                "title": "Check that the isPositron context key is set",
                "enablement": "isPositron"
            },
```

The PWB extension will be leveraging this context key soon, so we can also wait for that work to confirm the context key functionality: https://github.com/rstudio/rstudio-workbench-vscode-ext/issues/279